src/components/layout/UserHeader.tsx

```
@@ -1,221 +1,96 @@
+          <Link href="/" className="inline-block">
+            <Image
+              src="/logo.png"
+              width={1544}
+              height={380}
+              priority
+              alt="ddingdong"
+              className={cn(logoBase, logoMobile)}
+            />
+          </Link>
+          <button onClick={openModal} aria-label="Open navigation drawer">
+            <Icon name="list" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)